src/utils/utils.js

```diff
@@ -34,21 +34,6 @@ function promisify(context: any, method: any, ...args: any): Promise<mixed> {
   });
 }
 
-/**
- * @memberof utils/utils
- * @static
- */
-function endTruncateStr(str: any, size: number) {
```

Comment:
This will probably break tests as they are still expecting the string to be truncated manually. You'd need to remove tests that check if the text strings are truncated in order for this to pass.